docs/GettingStartedDocs/Contributors/JammySGX1FLCGettingStarted.md

```diff
@@ -0,0 +1,142 @@
+# Getting Started with Open Enclave in SGX1-FLC mode
```

Comment:
As this document is essentially the same between Ubuntu 22.04 and 20.04 (and other Ubuntu versions), you could add 22.04 to the current `SGX1FLCGettingStarted.md` doc, I think the request was to actually create a Ubuntu 22.04 specific version of [install_oe_sdk-Ubuntu_20.04.md](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md). Sorry for the miscommunication here. You will need to look through the Ansible files to see the differences between the two (e.g. libprotobuf)

Also as a side note, it would be more clear if this uses `Ubuntu 22.04` instead of `Jammy` as that is sometimes less well known. E.g. `SGX1FLCGettingStarted-Ubuntu22.04.md`